### PR TITLE
CRO: Attached the untagged instances to the email body

### PR DESCRIPTION
**Pull request by athiruma**
CloudResourceOrchestartion 
1. Send an attachment file with list of instances to the user for adding the tag TicketId

### CODE REVIEW

**Review by krishvoor — APPROVED** [review]
/lgtm

**Review by ebattat — COMMENTED** [review]
Please add PR description

File: cloud_governance/cloud_resource_orchestration/aws/ec2/monitor_tickets.py [hunk 1]
```diff
@@ -106,12 +109,20 @@ def __send_ticket_status_alerts(self, tickets: dict, ticket_status: str):
...
+                    if active_instances:
+                        for region, instances_list in active_instances.items():
+                            active_instances_ids = {region: [instance.get('InstanceId') for instance in instances_list]}
+                            to = user
+                            cc = self.__default_admins
+                            subject, body = self.__mail_message.cro_send_user_alert_to_add_tags(user=user, ticket_ids=ticket_ids)
+                            with tempfile.NamedTemporaryFile(mode='w', suffix='.json') as filename:
+                                filename.write(json.dumps(active_instances_ids))
+                                filename.flush()
+                                self.__postfix.send_email_postfix(to=to, cc=cc, subject=subject, content=body, mime_type='html', filename=filename.name)
```
Comment (ebattat) [inline cloud_governance/cloud_resource_orchestration/aws/ec2/monitor_tickets.py:125]: did u handle the case that no instances ?

Comment (athiruma) [reply cloud_governance/cloud_resource_orchestration/aws/ec2/monitor_tickets.py:125]: yes, I come to here after if found any instances.

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**